include/pistache/async.h

```diff
@@ -47,7 +47,7 @@ namespace Async {
 
     class BadAnyCast : public std::bad_cast {
     public:
-        virtual const char* what() const noexcept { return "Bad any cast"; }
+        virtual const char* what() const noexcept override { return "Bad any cast"; }
```

Comment:
If you add "override", you should remove "virtual".

I am surprised that cppcheck did not catch this one.

Reply:
done